test/distributed/fsdp/test_fsdp_state_dict.py

```diff
@@ -132,6 +132,23 @@ def forward(self, x):
         return self.outer(i + j)
 
 
+class TestDummyModel(torch.nn.Module):
```

Comment:
not necessary in this PR, but can we move it to torch/testing/_internal/common_fsdp.py?